lib/api/wasmedge.cpp

```diff
@@ -2174,15 +2172,9 @@ WASMEDGE_CAPI_EXPORT void WasmEdge_ImportObjectInitWASI(
       EnvVec.emplace_back(Envs[I]);
     }
   }
-  if (Dirs) {
-    for (uint32_t I = 0; I < DirLen; I++) {
-      DirVec.emplace_back(Dirs[I]);
-    }
-  }
   if (Preopens) {
     for (uint32_t I = 0; I < PreopenLen; I++) {
-      DirVec.emplace_back(std::string(Preopens[I]) + ":" +
-                          std::string(Preopens[I]));
+      DirVec.emplace_back(Preopens[I]);
```

Comment:
We should have two different cases:

1. `PATH1:PATH2`: map `PATH1` to `PATH2`.
2. `PATH`: map `PATH` to `PATH`

However, this section cannot present these two behaviour.

Reply:
This is actually handled in WASI:

https://github.com/WasmEdge/WasmEdge/blob/master/lib/host/wasi/environ.cpp#L58

Reply:
I don't think so. Looks like the `environ.cpp` only handles the `PATH1:PATH2` part. It doesn't create the mapping of second one.